lib/utils.js

```diff
@@ -35,6 +35,8 @@ const resolveImport = (context, node, { value, range }, currentPackage) => {
 const pathToImport = path => {
   if (path === '') return '.';
 
+  path = normalize(path).replace(/\\/g, '/');
+
   if (path[0] !== '.') return `./${path}`;
 
   return path;
```

Comment:
Thanks for this!
I'd prefer a new variable instead of mutating `path` here tho. Other than that LGTM
```suggestion
  const normalized = normalize(path).replace(/\\/g, '/');

  if (normalized[0] !== '.') return `./${normalized}`;

  return normalized;
```